packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -224,12 +224,14 @@ function TokenBalance({
   forToken,
   balance,
   on,
-  prefix = ''
+  prefix = '',
+  skipUSDCoverride = false
```

Comment:
I think this is a good start overall and gets the job done, but I'd like to simplify the approach even further.

- What if, instead of using `skipUSDCoverride` flag and propagating it's impact to `sanitizeTokenSymbol` functions as well, why don't we keep the change localised to this component only?
- Ie. Instead of using `skipUSDCoverride` we can use `overrideTokenSymbol` in the `TokenBalance` props and then the symbol used by this component can be `return overrideTokenSymbol ?? sanitizeTokenSymbol(forToken.symbol...`. 
- It bypasses the symbol generation locally and the impact within `sanitizeTokenSymbol` functions is avoided.

We can discuss this further as well but we need to keep in mind that it's just one case and the logic doesn't need to be reused or propagated to the core functions.